### PR TITLE
Update MDExpress to include 0.1.19 of TM, 0.1.19 of WM and 0.3.19 of IG

### PR DESCRIPTION
**Pull request by JHancox**
PR to address issue #129 - i.e. Latest builds of IG, WM & TM

### CODE REVIEW

File: deploy/monai-deploy-express/configs/informatics-gateway.json [hunk 1]
```diff
@@ -43,15 +70,14 @@
...
-      "tempStorageRootPath": "/incoming",
+      "localTemporaryStoragePath": "/payloads",
+      "remoteTemporaryStoragePath": "/incoming",
       "bucketName": "monaideploy",
       "storageRootPath": "/payloads",
       "temporaryBucketName": "monaideploy",
       "serviceAssemblyName": "Monai.Deploy.Storage.MinIO.MinIoStorageService, Monai.Deploy.Storage.MinIO",
       "watermarkPercent": 75,
       "reserveSpaceGB": 5,
-      "concurrentUploads": 4,
```
Comment (mocsharp) [inline deploy/monai-deploy-express/configs/informatics-gateway.json:54]: Please do keep `concurrentUploads`, this overrides the default value of 2 or 3 I think.

**Review by mocsharp — APPROVED** [review]

File: deploy/monai-deploy-express/configs/task-manager.json [hunk 2]
```diff
@@ -70,14 +65,14 @@
         "accessKey": "minioadmin",
         "accessToken": "minioadmin",
         "region": "eu-west-2",
-        "bucket": "monaideploy",
+        "bucket": "my-bucket",
```
[the remark this answers is not in the document]
Comment (JHancox) [reply deploy/monai-deploy-express/configs/task-manager.json]: Done - thanks

File: deploy/monai-deploy-express/configs/informatics-gateway.json [hunk 3]
```diff
@@ -1,11 +1,38 @@
 {
   "MonaiDeployAuthentication": {
-    "bypassAuthentication": true
+    "bypassAuthentication": true,
+    "openId": {
```
[the remark this answers is not in the document]
Comment (JHancox) [reply deploy/monai-deploy-express/configs/informatics-gateway.json]: Yep - removed

File: deploy/monai-deploy-express/configs/informatics-gateway.json (hunk 1 above)
Comment (JHancox) [reply deploy/monai-deploy-express/configs/informatics-gateway.json]: Added concurrentUploads in as suggested.

**Comment by JHancox** [comment]
Updated versions: Now 0.1.19 of WM & TM & 0.3.19 of IG

**Pull request merged**